src/index.ts

```diff
@@ -282,3 +282,59 @@ export const gt = (versionA: string | null, versionB: string | null): boolean =>
 export const lt = (versionA: string | null, versionB: string | null): boolean => {
 	return compare(versionA, versionB) < 0;
 };
+
+/**
+ * @summary Check if a version satisfies a range
+ * @name satisfies
+ * @public
+ * @function
+ *
+ * @description Return true if the version satisfies the range
+ *
+ * @param {string|null} version - The version to evaluate
+ * @param {string} range - A semver range string, see the [node-semver](https://github.com/npm/node-semver#ranges)
+ * docs for details
+ *
+ * @returns {boolean} - True if the version satisfies the range, false otherwise
+ *
+ */
+export const satisfies = (version: string | null, range: string) => {
+	if (!version) {
+		return false;
+	}
+
+	version = normalize(<string>version);
+
+	if (!semver.valid(version)) {
+		return false;
+	}
+
+	return semver.satisfies(version, range);
+};
+
+/**
+ * @summary Return the lowest version in the list that satisfies the range
+ * @name maxSatisfying
+ * @public
+ * @function
+ *
+ * @description Return the highest version in the list that satisfies the range, or null if none of them do.
```

Comment:
It would also be good to point out the semantics for equally highest values: if there are multiple equally high values, the first one found in the array is returned.